### PR TITLE
[PAN-3009] Fixes invalid JsonGetter, comment

### PR DESCRIPTION
**Pull request by josh-richardson**
## PR description
Fixes the following: 
The json getter on line 49 is mislabeled 
It should be changed from "privateFor" to "privacyGroupId"
Also, the comment above the class is not accurate

### CODE REVIEW

File: ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/privacy/PrivateTransactionGroupResult.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
 })
 /*
  The original deserialised private transaction sent via eea_sendRawTransaction
- This class is used if the original request was sent with privateFrom and privateFor
+ This class is used if the original request was sent with privacyGroupId
```
Comment (rain-on) [inline ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/privacy/PrivateTransactionGroupResult.java]: It'd be good to update the comment in PrivateTransactionLegacyResult as well (same reason).

Comment (josh-richardson) [reply ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/privacy/PrivateTransactionGroupResult.java]: This has now been addressed. Sorry for the delay.

File: ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/privacy/PrivateTransactionLegacyResult.java [hunk 2]
```diff
@@ -39,7 +39,7 @@
 })
 /*
  The original deserialised private transaction sent via eea_sendRawTransaction
- This class is used if the original request was sent with privateFrom and privateFor
+ This class is used if the original request was sent with privacyGroupId
```
Comment (iikirilov) [inline ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/privacy/PrivateTransactionLegacyResult.java]: the comment here was correct

Comment (rain-on) [reply ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/privacy/PrivateTransactionLegacyResult.java]: that is true - what was I thinking?

**Pull request merged**